lava-master/Dockerfile

```diff
@@ -1,79 +1,13 @@
-FROM bitnami/minideb:stretch
-
-RUN echo "deb http://ftp.fr.debian.org/debian/ stretch-backports main" >> /etc/apt/sources.list
-
-RUN apt-get update
-
-# Install debian packages used by the container
-# Configure apache to run the lava server
-# Log the hostname used during install for the slave name
-RUN echo 'lava-server   lava-server/instance-name string lava-docker-instance' | debconf-set-selections \
- && echo 'locales locales/locales_to_be_generated multiselect C.UTF-8 UTF-8, en_US.UTF-8 UTF-8 ' | debconf-set-selections \
- && echo 'locales locales/default_environment_locale select en_US.UTF-8' | debconf-set-selections
-
-# e2fsprogs is for libguestfs
-RUN DEBIAN_FRONTEND=noninteractive apt-get -y install \
- locales \
- postgresql \
- screen \
- sudo \
- wget \
- e2fsprogs \
- gnupg \
- vim
-
-RUN wget http://images.validation.linaro.org/production-repo/production-repo.key.asc \
- && apt-key add production-repo.key.asc \
- && echo 'deb http://images.validation.linaro.org/production-repo/ stretch-backports main' > /etc/apt/sources.list.d/lava.list \
- && apt-get clean && apt-get update
-
-RUN DEBIAN_FRONTEND=noninteractive apt-get -y install \
- qemu-system \
- qemu-system-arm \
- qemu-system-i386 \
- ser2net \
- u-boot-tools \
- python-setproctitle
-
-RUN if [ "$(uname -m)" = "x86_64" -o "$(uname -m)" = "x86" ] ;then apt-get -y install qemu-kvm ; fi
-
-#RUN service postgresql start \
-# && DEBIAN_FRONTEND=noninteractive apt-get -y install \
-# lava \
-# && /stop.sh
-
-# Install latest
-COPY scripts/build-lava /
-RUN bash /build-lava lava-dispatcher 2018.4 2018.4-1
-RUN /etc/init.d/postgresql start && \
- bash /build-lava lava-server 2018.4 2018.4-1 && \
- /etc/init.d/postgresql stop
+FROM baylibre/lava-master-docker-image:latest
```

Comment:
Please rename to baylibre/lava-master-base, and also add tags to Docker hub for lava versions.  Then use specific tag here.

Reply:
Actually, we don't need the base either.  Just baylibre/lava-master is enough.

Reply:
as discussed in meeting, changed to lava-master-base:latest